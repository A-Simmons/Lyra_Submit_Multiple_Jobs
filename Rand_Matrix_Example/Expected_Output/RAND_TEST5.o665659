WARNING: ignoring environment value of R_HOME

-----
PBS Job 665659.pbs
CPU time  : 00:00:00
Wall time : 00:00:02
Mem usage : 30032kb
