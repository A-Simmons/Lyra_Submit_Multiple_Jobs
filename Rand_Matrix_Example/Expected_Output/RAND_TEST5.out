
R version 3.2.4 (2016-03-10) -- "Very Secure Dishes"
Copyright (C) 2016 The R Foundation for Statistical Computing
Platform: x86_64-pc-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

  Natural language support but running in an English locale

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.

> args<-commandArgs(TRUE)
> 
> # Set some defaults
> seed <- 1
> n <- 10
> m <- 10
> 
> # Replace defaults with arguments if they exist
> nargs = length(args)
> if (nargs >= 1) {
+   seed <- eval( parse(text=args[1]))
+   if (nargs >= 2) {
+     n <- eval( parse(text=args[2]))
+     if (nargs >= 3) {
+       m <- eval( parse(text=args[3]))
+     }
+   }
+ }
> set.seed(seed)
> 
> 
> print(c(seed, n, m))
[1] 4000    6    1
> print(matrix(rexp(200, rate=.1),nrow=n,ncol=m))
            [,1]
[1,] 14.26555347
[2,]  9.95167768
[3,] 13.20747773
[4,]  0.07777042
[5,]  7.29367171
[6,]  6.63075747
Warning message:
In matrix(rexp(200, rate = 0.1), nrow = n, ncol = m) :
  data length [200] is not a sub-multiple or multiple of the number of rows [6]
> 
> proc.time()
   user  system elapsed 
  0.196   0.024   0.221 
